BennyTordrupVisma/Week2/Ass3/script.js

```diff
@@ -0,0 +1,340 @@
+var Message = React.createClass({
+	getInitialState: function() {
+		return {
+			listSelectionVisible : false
+		}
+	},
+	
+	render: function() {
+		return (
+			<div>
+					{this.props.message.isArchived 
+						? <div>
```

Comment:
For rather large jsx blocks, such as these ones, I would have had 2 local variables which stored a reference to the jsx. I think it would have made the code more readable:

```jsx
{this.props.message.isArchived ? archivedMessage : nonarchivedMessage}
```